pkg/controller/handler.go

```diff
@@ -170,11 +171,12 @@ func (h *handler) CreateServiceBinding(in *servicecatalog.Binding) (*servicecata
 	// Assign UUID to binding.
 	in.Spec.OSBGUID = uuid.NewV4().String()
 
+	// TODO: uncomment parameters line once parameters types are refactored.
```

Comment:
I confirmed with people that the existing stuff doesn't rely on Parameters or Metadata today.  Going to re-introduce these lines in a follow-up (immediate) PR.